src/python/dart/model/subscription.py

```diff
@@ -77,6 +78,12 @@ class SubscriptionElementState(object):
     CONSUMED = 'CONSUMED'
 
 
+class NudgeElementState(object):
```

Comment:
You don't use this in your changes. Also the current states are
AVAILABLE
BATCHED